korman/properties/modifiers/render.py

```diff
@@ -49,7 +49,8 @@ class PlasmaBlendMod(PlasmaModifierProperties):
                                 items=[("AUTO", "(Auto)", "Let Korman decide when to render this object."),
                                        ("OPAQUE", "Before Avatar", "Prefer for the object to draw before the avatar."),
                                        ("FRAMEBUF", "Frame Buffer", "Prefer for the object to draw after the avatar but before other blended objects."),
-                                       ("BLEND", "Blended", "Prefer for the object to draw after most other geometry in the blended pass.")],
+                                       ("BLEND", "Blended", "Prefer for the object to draw after most other geometry in the blended pass."),
+                                       ("LATE", "Late", "Prefer for the object to draw late in the blended pass.")],
```

Comment:
When is "late" in this context?  The other options are more explicit about when the object is drawn.

Reply:
it translates to an 8 for the major bit in the render pass

Reply:
"Late" happens after all other alpha-blended objects, and is being used (in this particular use case) to match the flags that Cyan uses on GZ Markers.

The full render-level behaviour: https://github.com/H-uru/Plasma/blob/3b2788c23f6437f2fd81a18a451173b73a92319a/Sources/Plasma/CoreLib/plRenderLevel.h#L49-L79

Reply:
For the sake of the description, is that at the very end of the render pass, after the end, at the end of all render passes...?

Reply:
"Prefer for the object to draw after all other alpha-blended objects." might be more descriptive then.